src/common/services/api/designations.service.spec.js

```diff
@@ -209,6 +209,20 @@ describe('designation service', () => {
         })
       self.$httpBackend.flush()
     })
+
+    it('should ignore givingLinks without names or urls', () => {
+      const response = angular.copy(designationResponse)
+      response['jcr:content'].givingLinks.item1 = { 'jcr:primaryType': 'nt:unstructured' }
```

Comment:
We can test my question above by also adding an `item2` that is valid and ensuring it is in the response.